### PR TITLE
fix(one-app-runner): fix runner not reading package.json correctly

### PR DESCRIPTION
**Pull request by JamesSingleton**
This also wraps NO_PROXY, HTTP_PROXY and HTTPS_PROXY values with quotes

### CODE REVIEW

File: packages/one-app-runner/src/startApp.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ module.exports = async function startApp({
   };
 
   const command = `docker pull ${appDockerImage} && docker run -t -p 3000-3005:3000-3005 -e NODE_ENV=development ${generateEnvironmentVariableArgs(envVars)} ${generateModuleMountsArgs(modulesToServe)} ${appDockerImage} /bin/sh -c "${generateServeModuleCommands(modulesToServe)} ${generateSetMiddlewareCommand(parrotMiddlewareFile)} ${generateSetDevEndpointsCommand(devEndpointsFile)} node lib/server/index.js --root-module-name=${rootModuleName} --module-map-url=${moduleMapUrl} ${generateUseMocksFlag(parrotMiddlewareFile)}"`;
+  console.log(command);
```
Comment (infoxicator) [inline packages/one-app-runner/src/startApp.js:81]: `unexpected console.log` or expected `console.log`?

**Comment by JamesSingleton** [comment]
> How did you test these changes?

@infoxicator, I tested this by both running `npm pack` and installing the `.tgz` file within a v5 module as well as running `node ./bin/one-app-runner` within `packages/one-app-runner`

File: packages/one-app-runner/bin/one-app-runner.js [hunk 2]
```diff
@@ -25,8 +26,12 @@ const createYargsConfig = () => {
 
   const oneAppRunnerConfig = (packageJson['one-amex'] && packageJson['one-amex'].runner) || {};
 
+  if (Object.keys(oneAppRunnerConfig).length > 0) {
+    yargs.config(oneAppRunnerConfig);
+  }
+
   // eslint-disable-next-line global-require
```
Comment (JAdshead) [inline packages/one-app-runner/bin/one-app-runner.js:33]: this can be removed

**Comment by JamesSingleton** [comment]
> one-app-runner starts but we are getting the following warning on every module map poll
> `warn: one-app-dev-cdn error loading module map from undefined: RequestError: getaddrinfo ENOTFOUND undefined`

@infoxicator how are you testing this? I have both ran the bin file individually as well as packed the library into a tgz.

**Comment by infoxicator** [comment]
> > one-app-runner starts but we are getting the following warning on every module map poll
> > `warn: one-app-dev-cdn error loading module map from undefined: RequestError: getaddrinfo ENOTFOUND undefined`
> 
> @infoxicator how are you testing this? I have both ran the bin file individually as well as packed the library into a tgz.

yep library as tgz, I am using a docker hub image that has one of the most recent versions of the app:  ```"dockerImage": "infoxicator/one-app-dev:v5.0.18-rc1"``` could be something in one app that is causing the warning

**Comment by JamesSingleton** [comment]
> Could you please track the warning, we should look at that separately.

1fbfd31 actually fixes this warning.

File: packages/one-app-runner/src/startApp.js [hunk 3]
```diff
@@ -77,7 +77,14 @@ module.exports = async function startApp({
     return command;
   };
 
-  const command = `docker pull ${appDockerImage} && docker run -t -p 3000-3005:3000-3005 -e NODE_ENV=development ${generateEnvironmentVariableArgs(envVars)} ${generateModuleMountsArgs(modulesToServe)} ${appDockerImage} /bin/sh -c "${generateServeModuleCommands(modulesToServe)} ${generateSetMiddlewareCommand(parrotMiddlewareFile)} ${generateSetDevEndpointsCommand(devEndpointsFile)} node lib/server/index.js --root-module-name=${rootModuleName} --module-map-url=${moduleMapUrl} ${generateUseMocksFlag(parrotMiddlewareFile)}"`;
+  const generateModuleMap = () => {
+    if (moduleMapUrl) {
```
Comment (infoxicator) [inline packages/one-app-runner/src/startApp.js:81]: ternary operator anyone? 😂JKD I'm not fussed about that...

Comment (JAdshead) [reply packages/one-app-runner/src/startApp.js:81]: @JamesSingleton 
```suggestion
    return moduleMapUrl ? `--module-map-url=${moduleMapUrl}` : '';
```

**Pull request merged**